src/utilities.client.js

```diff
@@ -342,13 +346,13 @@ exports.unescapeFromBson = function (obj, clone) {
 exports.isBadForMongo = function (obj) {
     let isBad = false;
     try {
-        traverseObject(obj, false, (key) => {
+        traverseObject(obj, false, (key, value) => {
             const escapedKey = escapePropertyName(key);
             if (key !== escapedKey) {
                 isBad = true;
                 throw new Error();
             }
-            return key;
+            return [key, value];
```

Comment:
Since the return type of this function switches from `string` to `array`, shouldn't we make this a breaking change and bump major instead? I also need to bump major in #59 so we could publish 4.0 with both changes.